test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1089,6 +1089,29 @@ class test_ucp_wireup_unified : public test_ucp_wireup {
         return false;
     }
 
+    bool ep_iface_has_caps(ucp_worker_h worker, const std::string& tl,
+                           uint64_t caps)
+    {
+        ucp_context_h ctx = worker->context;
+
+        for (unsigned i = 0; i < worker->num_ifaces; ++i) {
+            ucp_worker_iface_t *wiface = worker->ifaces[i];
+            uct_iface_h iface          = wiface->iface;
+            uct_iface_attr_t iface_attr;
+
+            char* name = ctx->tl_rscs[wiface->rsc_index].tl_rsc.tl_name;
+            if (strcmp(name, tl.c_str())) {
+                continue;
+            }
+
+            ucs_status_t status = uct_iface_query(iface, &iface_attr);
+            ASSERT_UCS_OK(status);
```

Comment:
use wiface->iface_attr (and remove local var 'iface')

Reply:
ok, done